rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java

```diff
@@ -139,6 +137,24 @@ void test_getBtcHeightWhenPegoutTxIndexActivates(BridgeConstants bridgeConstants
         int btcHeightWhenPegoutTxIndexActivates = bridgeConstants.getBtcHeightWhenPegoutTxIndexActivates();
 
         // assert
-        Assertions.assertEquals(expectedValue, btcHeightWhenPegoutTxIndexActivates);
+        assertEquals(expectedValue, btcHeightWhenPegoutTxIndexActivates);
+    }
+
+    private static Stream<Arguments> getPegoutTxIndexGracePeriodInBtcBlocksArgProvider() {
+        return Stream.of(
+            Arguments.of(BridgeMainNetConstants.getInstance(), 4_320),
+            Arguments.of(BridgeTestNetConstants.getInstance(), 4_320),
```

Comment:
Can these values `4_320`, etc., be used from the constants classes? I see they are defined in `BridgeDevNetConstants`. I think it would be more readable this way.
Or at least from a constant in this test file if this value would change for tests.

Reply:
Gotcha.